### PR TITLE
Mds 2399 contact verification

### PR DESCRIPTION
**Pull request by mmumby**
- Ability to match NoW contacts to contacts in CORE
- Ability to edit those contacts later
Screen Shot 2020-10-21 at 1 41 39 PM
Screen Shot 2020-10-21 at 2 39 10 PM

### CODE REVIEW

File: services/core-api/app/api/now_applications/resources/now_application_import_resource.py [hunk 1]
```diff
@@ -61,6 +67,26 @@ def post(self, application_guid):
...
+            for contact in contacts:
+
+                now_party = Party.find_by_party_guid(contact['party_guid'])
+                if not now_party:
+                    raise NotFound('No party found for party with guid ' + contact['party_guid'])
+
+                mine_party_appt_type = MinePartyAppointmentType.find_by_mine_party_appt_type_code(
+                    contact['mine_party_appt_type_code'])
+                if not mine_party_appt_type:
+                    raise NotFound('No mine party appointment type found for type with code ' +
```
Comment (exzizt) [inline services/core-api/app/api/now_applications/resources/now_application_import_resource.py:80]: I think we prefer the `f''` syntax for concatenating strings but I'm not gonna nitpick that hard!

File: services/core-web/src/components/common/RenderLargeSelect.js [hunk 2]
```diff
@@ -18,14 +18,19 @@ const propTypes = {
...
     touched: PropTypes.bool,
     error: PropTypes.string,
     warning: PropTypes.string,
   }).isRequired,
   handleSearch: PropTypes.func,
   handleSelect: PropTypes.func,
+  handleFocus: PropTypes.func,
 };
 
 const doNothing = () => {};
```
Comment (exzizt) [inline services/core-web/src/components/common/RenderLargeSelect.js:36]: image

Comment (mmumby) [reply services/core-web/src/components/common/RenderLargeSelect.js:36]: focus, Luke, focus!

File: services/core-web/src/components/noticeOfWork/applications/verification/ApplicationStepOne.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import React, { Component } from "react";
 import { bindActionCreators } from "redux";
+import { submit } from "redux-form";
```
Comment (exzizt) [inline services/core-web/src/components/noticeOfWork/applications/verification/ApplicationStepOne.js:3]: Is this really being used anywhere?

Comment (mmumby) [reply services/core-web/src/components/noticeOfWork/applications/verification/ApplicationStepOne.js:3]: nope, removed! good catch

File: services/core-web/src/components/common/RenderLargeSelect.js (hunk 2 above)
Comment (exzizt) [reply services/core-web/src/components/common/RenderLargeSelect.js:36]: It's the `doNothing = () => {};`!12!@!1@!!

**Review by exzizt — APPROVED** [review]
Great group effort on this one, Ryan and Meredith! I don't have any problems with anything. Looks great to me! 

image

**Review by exzizt — APPROVED** [review]

**Pull request merged**